src/components/AssetSelector.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from 'styled-components';
+import { OptionProps } from 'react-select';
+
+import { translateRaw } from '@translations';
+import { Asset, ISwapAsset, TSymbol, TUuid } from '@types';
+import { AssetIcon, Typography, Selector } from '@components';
+
+const SContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 14px 15px 14px 15px;
+`;
+
+export interface ItemProps {
+  uuid: TUuid;
+  symbol: TSymbol;
+  name?: string;
+  onClick?(): void;
+}
+
+export function AssetSelectorItem({ uuid, symbol, name, onClick }: ItemProps) {
+  return (
+    <SContainer
+      {...(onClick ? { onPointerDown: onClick } : null)}
+      data-testid={`asset-dropdown-option-${symbol}`}
+    >
+      <AssetIcon uuid={uuid} size={'1.5rem'} />
+      <Typography bold={true} value={symbol} style={{ marginLeft: '10px' }} />
+      {name && <span>&nbsp; - &nbsp;</span>}
+      <Typography value={name} />
+    </SContainer>
+  );
+}
+
+const Label = styled.label`
+  font-size: 18px;
+  width: 100%;
+  line-height: 1;
+  text-align: left;
+  font-weight: normal;
+  margin-bottom: 9px;
+  color: ${(props) => props.theme.text};
+`;
+
+const Wrapper = styled('div')`
+  width: ${(props: { fluid: boolean }) => (props.fluid ? '100%' : 'default')};
+  min-width: 175px;
+  .asset-dropdown-item {
+    padding-top: 11px;
+    padding-bottom: 11px;
+  }
+`;
+
+export interface AssetSelectorProps<T> {
+  inputId?: string;
+  assets: T[];
+  selectedAsset: T | null;
+  showOnlySymbol?: boolean;
+  disabled?: boolean;
+  fluid?: boolean;
+  searchable?: boolean;
+  label?: string;
+  onSelect(option: T): void;
+}
+
+export type TAssetOption = Asset | ISwapAsset;
+
+function AssetSelector({
```

Comment:
There is a gap at the top and bottom of the selector options:

Old:
![image](https://user-images.githubusercontent.com/1561200/88053545-439cca00-cb5c-11ea-905d-beac0c3cc297.png)


New:
![image](https://user-images.githubusercontent.com/1561200/88053514-3a136200-cb5c-11ea-9a4f-7f601121a190.png)

Reply:
Done.
Nice catch 👍